web/router.ex

```diff
@@ -11,9 +11,7 @@ defmodule RemoteRetro.Router do
     plug :fetch_flash
     plug :protect_from_forgery
     plug :put_secure_browser_headers
-    if Application.get_env(:remote_retro, :add_mock_user_to_session) do
-      plug :add_mock_user_to_session
-    end
+  
```

Comment:
pulled the branch, and you're right. Not sure how google is posting to localhost, but I won't question it.

One drawback here is that this adds a network dependency for devs who don't have a user on session. In truth, there's *already* a network dependency due to using a CDN-hosted semantic-ui stylesheet, but that's something I was hoping to kill, and I think @Zanadar  [was too](https://github.com/stride-nyc/remote_retro/issues/74). Let's connect on monday and discuss the pluses and minuses of actually making devs authenticate.